samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -99,7 +117,10 @@ - (void)viewDidLoad {
     _bLabel = [[UILabel alloc] init];
     _boxLabel = [[UILabel alloc] init];
 
-    _img = [UIImage imageNamed:[NSString stringWithFormat:@"photo%d.jpg", _accelerateImageNumber]];
+    UIImage* image = [UIImage imageNamed:[NSString stringWithFormat:@"photo%d.jpg", _accelerateImageNumber]];
+    CGImageRef convertedImage = _CGImageCreate32bppCopy(image.CGImage);
+    _img = [UIImage imageWithCGImage:convertedImage];
+    CGImageRelease(convertedImage);
```

Comment:
I like it, but I'll defer to what eh have here. Accelerate's integration tests are not really great, and they could use a lot of cleanup. Setting the image twice isn't even necessary here, so we should evaluate just not doing it.